libs/sdk-ui-semantic-search/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+// (C) 2020-2024 GoodData Corporation
+module.exports = {
+    parser: "@typescript-eslint/parser",
+    plugins: ["react-hooks", "prettier", "sonarjs", "eslint-plugin-tsdoc"],
+    extends: [
+        "@gooddata",
```

Comment:
Pls can you verify (nejsem si jisty) that eslint is set check hooks and has set it as error (warnings are deafult)

        "react-hooks/rules-of-hooks": 2,
        "react-hooks/exhaustive-deps": 2,